src/sql/workbench/contrib/notebook/browser/outputs/notebookMarkdown.ts

```diff
@@ -245,4 +249,27 @@ export class NotebookMarkdownRenderer {
 	setNotebookURI(val: URI) {
 		this._notebookURI = val;
 	}
+
+	/**
+	 * The following is a sample cell attachment from JSON:
+	 *  "attachments": {
+	 *     "test.png": {
+	 *        "image/png": "iVBORw0KGgoAAAANggg==="
+	 *     }
+	 *  }
+	 */
+	findAttachmentIfExists(href: string, cellAttachments: { [key: string]: { [key: string]: string } }): string {
+		if (href.startsWith('attachment:') && cellAttachments) {
```

Comment:
Where does `attachment:` come from? The doc comment doesn't seem to say anything about that